crates/essential-rest-client/src/main.rs

```diff
@@ -11,28 +11,14 @@ use std::{path::PathBuf, str::FromStr};
 #[command(version, about, long_about = None)]
 /// Essential REST Client
 struct Cli {
-    /// The endpoint of node to bind to.
-    #[arg(long)]
-    node_address: Option<String>,
-    /// The endpoint of builder to bind to.
-    #[arg(long)]
-    builder_address: Option<String>,
     #[command(subcommand)]
-    commands: Commands,
+    command: Command,
+    address: String,
```

Comment:
I would actually add this to each sub command. Then when you do `list-blocks --help` it will actually show you what's required.

Reply:
Oh yeah that was @mitchmindtree's suggestion too. Updated